src/gui/torrentmodel.cpp

```diff
@@ -36,9 +36,15 @@
 
 #include "base/bittorrent/session.h"
 #include "base/bittorrent/torrenthandle.h"
+#include "base/preferences.h"
 #include "base/torrentfilter.h"
+#include "base/unicodestrings.h"
 #include "base/utils/fs.h"
+#include "base/utils/misc.h"
+#include "base/utils/string.h"
 #include "torrentmodel.h"
+#include "transferlistdelegate.h"
+#include "transferlistwidget.h"
```

Comment:
Oops. You've created wrong dependencies! The model MUST NOT in any way depend on the widgets that use it (and their delegates). You require redesign your code.